### PR TITLE
add local files note to gitrepositories.md (fixes #715)

### LINKED ISSUE #715: Tell the user where cloned git repositories are saved in gitrepositories.md
### Problem
In Step 5, the user is told how to clone a repository to their computer using the git clone command. However, the user is never told where the actual repo files are saved to.

 ### Steps to reproduce the problem
Go to https://treehouses.io/#!./pages/vi/gitrepositories.md

 ### Screenshots
N/A

 ### Proposed solution
In Step 5, add a note indicating that the git repo local files are saved in `C:\Users\{reponame}` by default if they decide to use the `git clone` command.

### PR DESCRIPTION
**Pull request by leosp123**
Fixes #715

### Description
Adds a note, telling the user where `git clone` files are saved.

### Raw.Githack preview link
https://raw.githack.com/leosp123/leosp123.github.io/localfilesave/#!pages/vi/gitrepositories.md

### CODE REVIEW

**Review by anastasia21112 — APPROVED** [review]

**Review by sarahrams — COMMENTED** [review]
I am a bit skeptical about this. I don't know what @dogi thinks ;)

**Comment by anthonyhvo12** [comment]
The C:\Users folder only applies to Windows. The folder on Mac systems will be different.

**Comment by leosp123** [comment]
I can make a note stating that this applies to Windows users only. I do not know where the default folder is on Mac.

File: pages/vi/gitrepositories.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Use the following commands in your command line, but refer back to the descripti
 2. Copy the HTTPS or SSH link from your repository on the GitHub site
 3. On the command line, type `git clone *paste your HTTPS or SSH link here*`
 
+The local files will be saved at `C:\Users\{your-repo-here}` if you decide to use the `git clone` command.
```
Comment (dogi) [inline pages/vi/gitrepositories.md:27]: despite it only being for windows it is also kinda wrong
my git bash opens in `C:\Users\<username>\` where git clone then is leaving the repo

**Comment by dogi** [comment]
@leosp123 I am closing this one

**Pull request closed without merging**